docs/content/3.manual/manual.yml

```diff
@@ -271,17 +271,24 @@ sections:
             input: '"Hello, world!"'
             output: ['"Hello, world!"']
 
-      - title: "`.foo`, `.foo.bar`"
+      - title: "Object Identifier-Index: `.foo`, `.foo.bar`"
```

Comment:
Hmm, well, I want users to understand why `.foo` doesn't work for keys like "foo$bar.baz"...  I want the reason to jump at them.  Maybe there's no hope!  :)